Dockerfile

```diff
@@ -1 +1,185 @@
+#
+# This one houses the main git clone
+#
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest as src
 
+#
+# Basic Parameters
+#
+ARG ARCH="amd64"
+ARG OS="linux"
+ARG VER="0.40.0"
+ARG PKG="google_cadvisor"
+ARG SRC="https://github.com/google/cadvisor.git"
+
+#
+# Some important labels
+#
+LABEL ORG="Armedia LLC"
+LABEL MAINTAINER="Armedia Devops Team <devops@armedia.com>"
+LABEL APP="Google cAdvisor"
+LABEL VERSION="${VER}"
```

Comment:
I just realized I don't think we need to add labels for intermediate images. What do you think?

Reply:
I thought of that repeatedly on the Grafana one that has a bunch ... then I figured I'd just leave them in for consistency's sake, _just in case_.

They do no harm, and adding them helps for form a solid habit of maintaining them...

Reply:
OK